src/storage/src/hummock/sstable/mod.rs

```diff
@@ -353,7 +372,8 @@ impl SstableMeta {
             .map(| tombstone| 16 + tombstone.start_user_key.encoded_len() + tombstone.end_user_key.encoded_len())
             .sum::<usize>()
             + 4 // bloom filter len
-            + self.bloom_filter.len()
```

Comment:
Prior to this PR, we use `bloom_filter.len()` to calculate the bloom filter size but it is no longer the case. Please find all the usage of `bloom_filter.len()` and change them accordingly. I did a quick search and we need to change them in `builder.rs` and `sst_dump.rs`.